src/analysis/souffle/sensitivity_analysis.dl

```diff
@@ -0,0 +1,132 @@
+#ifndef SENSITIVITY_ANALYSIS_DL_
+#define SENSITIVITY_ANALYSIS_DL_
+//this file defines the calculations for sensitivity analysis 
+//to be used in dp calculations
+
+#include "src/analysis/souffle/taint.dl"
+
+//one isSqlOperator fact per each sql operator, currently supporting one output
+//referring to results as 'value' (sqlValue declared below) as per the discussion in the doc 
+.decl isSqlOperator(sqlOperator: Operation)
```

Comment:
Why does `isSqlOperator` contain an `Operation` rather than an `Operator`?

Reply:
the decision came after a discussion with @bgogul. AFAIK, all input to the datalog facts has to be formatted as an Operation. The name 'sql operator' was also discussed in the google doc and I switched to operator after @bgogul suggested it. If I misunderstood something, I'll be happy to revert/make more changes.

Reply:
May be call it `isSqlOperation`?

Reply:
Done.